src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/SHA256.cs

```diff
@@ -36,21 +30,22 @@ public Implementation()
                 HashSizeValue = _hashProvider.HashSizeInBytes * 8;
             }
 
-            protected sealed override void HashCore(byte[] array, int ibStart, int cbSize)
-            {
+            protected sealed override void HashCore(byte[] array, int ibStart, int cbSize) =>
                 _hashProvider.AppendHashData(array, ibStart, cbSize);
-            }
 
-            protected sealed override byte[] HashFinal()
-            {
-                return _hashProvider.FinalizeHashAndReset();
-            }
+            protected override void HashCore(ReadOnlySpan<byte> source) =>
```

Comment:
>if you don't like my agree-to-disagree comment, then what exactly do you suggest we do?

That depends on what you mean by "agree-to-disagree."

If you mean "I don't want to do that work for this PR because I like it the way it is", that's fine - I don't work much on crypto stuff anymore so I'm not interested in pushing harder on that.

If you mean "You will not use extra 'sealed' anywhere, and and you will remove them even in sections of the codebase that have maintained this discipline for far longer than corefx has been around", then yes, expect very strong pushback. (That's also not "agree-to-disagree".)